### ISSUE TITLE
Korean Greywyvern input not working as expected

### ISSUE THREAD

**Issue by fariskas** [Mottie/Keyboard#624]
Implemented the keyboard into a project only to realise it doesn't input correctly.
It also doesnt work correctly from the example shown here:
https://mottie.github.io/Keyboard/docs/layouts2.html

The lower '받짐' joining letters dont work.
For example:
질문 becomes 지ᄅ무ᄂ
간다 become 가ᄂ다  

I also think this issue opened previously is similar to what i have.
https://github.com/Mottie/Keyboard/issues/539

**Comment by Mottie** [comment]
Hi @fariskas!

Thanks for reporting the problem. Yes, it does appear to be a duplicate. Any chance you know how to fix the problem?

**Comment by fariskas** [comment]
no. unfortunately i dont. i dont even speak korean actually. just using it for a client

**Comment by JarredSteenvoorden** [comment]
@Mottie @fariskas  we also had the same problem and needed to implement the following javascript engine to convert English keypresses to Korean taking care of the joining characters.

http://www.cs.rochester.edu/u/ysong/korean.html

Perhaps this is something that could be integrated into the Mottie keyboard library?
